contrib/android/Dockerfile

```diff
@@ -154,23 +165,29 @@ RUN chown --recursive ${USER} ${WORK_DIR} ${ANDROID_SDK_HOME}
 RUN chown ${USER} /opt
 USER ${USER}
 
+# venv, VIRTUAL_ENV is used by buildozer to indicate a venv environemnt
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv ${VIRTUAL_ENV}
+ENV PATH="${VIRTUAL_ENV}/bin:${PATH}"
 
 COPY contrib/deterministic-build/requirements-build-base.txt /opt/deterministic-build/
 COPY contrib/deterministic-build/requirements-build-android.txt /opt/deterministic-build/
-RUN python3 -m pip install --no-build-isolation --no-dependencies --user \
+RUN /opt/venv/bin/python3 -m pip install --no-build-isolation --no-dependencies \
     -r /opt/deterministic-build/requirements-build-base.txt
-RUN python3 -m pip install --no-build-isolation --no-dependencies --no-binary :all: --user \
+RUN /opt/venv/bin/python3 -m pip install --no-build-isolation --no-dependencies --no-binary :all: \
     -r /opt/deterministic-build/requirements-build-android.txt
 
 # install buildozer
 RUN cd /opt \
     && git clone https://github.com/kivy/buildozer \
     && cd buildozer \
     && git remote add sombernight https://github.com/SomberNight/buildozer \
+    && git remote add accumulator https://github.com/accumulator/buildozer \
     && git fetch --all \
-    # commit: from branch sombernight/electrum_20210421 (note: careful with force-pushing! see #8162)
-    && git checkout "6f03256e8312f8d1e5a6da3a2a1bf06e2738325e^{commit}" \
-    && python3 -m pip install --no-build-isolation --no-dependencies --user -e .
+    # commit: from branch master (note: careful with force-pushing! see #8162) \
+    # no, this is master
```

Comment:
I guess it is not master afterall! :P

Reply:
it was for a long time, but good catch :)